packages/storybook/package.json

```diff
@@ -15,13 +15,15 @@
     "@babel/core": "^7.15.8",
     "@department-of-veterans-affairs/formation": "^7.0.1",
     "@mdx-js/react": "^1.6.22",
-    "@storybook/addon-actions": "^6.4.19",
-    "@storybook/addon-docs": "^6.4.19",
-    "@storybook/addon-essentials": "^6.4.19",
-    "@storybook/addon-links": "^6.4.19",
-    "@storybook/builder-webpack5": "^6.4.19",
-    "@storybook/manager-webpack5": "^6.4.19",
-    "@storybook/react": "^6.4.19",
+    "@storybook/addon-a11y": "^6.5.6",
+    "@storybook/addon-actions": "^6.5.6",
+    "@storybook/addon-docs": "^6.5.6",
+    "@storybook/addon-essentials": "^6.5.6",
+    "@storybook/addon-links": "^6.5.6",
+    "@storybook/addon-storysource": "^6.5.6",
+    "@storybook/builder-webpack5": "^6.5.6",
+    "@storybook/manager-webpack5": "^6.5.6",
+    "@storybook/react": "^6.5.6",
```

Comment:
Opportunistic update of Storybook to 6.5.6. I ran into some issues getting addon-a11y to work and updating to the latest Storybook resolved them.